proto/assessment.proto

```diff
@@ -39,33 +39,30 @@ option go_package = "api/assessment";
 // Representing the link between orchestrator and discovery: Assessing evidences
 // from discovery and sending results to orchestrator
 service Assessment {
-  // Triggers the assessment. Part of the private API,
-  // not exposed as REST.
+  // Triggers the assessment. Part of the private API. Not exposed as REST.
   rpc TriggerAssessment(TriggerAssessmentRequest)
       returns (google.protobuf.Empty) {}
 
-  // TODO(all): Part of public API because external entities (mainly
-  // discoveries) can use it? List the latest set of assessment results. Part of
-  // the public API, also exposed as REST
+  // List all assessment results. Part of the public API, also exposed as REST.
   rpc ListAssessmentResults(ListAssessmentResultsRequest)
       returns (ListAssessmentResultsResponse) {
     option (google.api.http) = {
-      post : "/v1/assessment/results"
+      get : "/v1/assessment/results"
       response_body : "*"
     };
   }
 
-  // Assesses the evidence sent by discovery. Part of the public API,
-  // also exposed as REST
+  // Assesses the evidence sent by the discovery. Part of the public API, also exposed as REST.
+  // TODO(all): Evidence instead of AssessEvidenceRequest?
```

Comment:
If request only contains one item/message, use it directly instead of creating a wrapper #googleapiconventions? @oxisto  @garuppel

Reply:
Doesn't this only relate to responses?

Reply:
> 

The following is from the Google API design guide (also in the proto README.md).

"For custom methods, they should have their own XxxResponse messages even if they are empty, because it is very likely their functionality will grow over time and need to return additional data." ([Link](https://cloud.google.com/apis/design/design_patterns#empty_responses))

Reply:
> 

Regarding the request message I didn´t find any information, but most examples have a dedicated request message with one field.

Reply:
[Request and response messages (API designe guide)](https://cloud.google.com/apis/design/naming_convention#request_and_response_messages):
> The request and response messages for RPC methods should be named after the method names with the suffix Request and Response, respectively, unless the method request or response type is:
    - an empty message (use google.protobuf.Empty),
    - a resource type, or
    - a resource representing an operation 

But see also [Empty Responses (API designe guide)](https://cloud.google.com/apis/design/design_patterns#empty_responses):
> For custom methods, they should have their own XxxResponse messages even if they are empty, because it is very likely their functionality will grow over time and need to return additional data

Thus, I would say XxxResponse for response messages and resource type for request if possible. But we can do that in another PR as well; since such changes would affect the code directly. And we want to keep our PRs small :)

Reply:
I would use response and request messages as long as we don´t know if they are changing again. So, we do not have to rebuild everything if we are changing the messages.  :)

Reply:
We do not use XxxRequest messages in `rpc AssessEvidences(stream Evidence)` and in the `evidence.proto` file. Should I change them in this PR?

Reply:
I would change it in this PR. 

But I would wait for Christian's opinion before you change it back again :smile:

Reply:
Let's discuss this briefly in our JF today. I personally would follow the example here: https://cloud.google.com/apis/design/naming_convention#method_names. Which basically uses `Request` all the the time (probably for forwards compatibility) but omits `Response` for single resource, such as `Book` in the example for convenience.